hlxsites/merative2
# feat(breadcrumb): Add breadcrumb block fragment

## Issue

Fixes https://jira.sdlc.merative.com/browse/MERATIVE-790

## Description

**New**

- Add new block `blocks/breadcrumb` - The content of this block is currently being managed in the `/fragment/breadcrumbs` directory of SharePoint.
- Breadcrumb can be added to a page using the Metadata table block (see attached). Additionally, you can optionally hide the brand logo/image for certain pages that already have the band logo in the leadspace.

![Screenshot 2023-09-06 at 4 26 49 PM](https://github.com/hlxsites/merative2/assets/1815714/bc252658-b190-4989-bd1b-a3f235740406)

![Screenshot 2023-09-06 at 4 27 43 PM](https://github.com/hlxsites/merative2/assets/1815714/b7af26f5-3144-4afa-946f-b3edac172659)

## Design Specs

- Figma Link - https://www.figma.com/file/SPC6Jyf0RA8K29vTNkFXwg/Contact-us-%2B-success-message?type=design&node-id=943%3A10868&mode=design&t=EEDWQ99E5AjJyma3-1

## Test URLs

### MDX Sub Product page with a breadcrumb
![Screenshot 2023-09-06 at 4 39 09 PM](https://github.com/hlxsites/merative2/assets/1815714/30b332e3-ef39-4753-a289-9609240d62fc)

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/clinical-decision-support/pharmacy-inventory-management
- After (Changes from this PR): https://feat-breadcrumb--merative2--proeung.hlx.page/clinical-decision-support/pharmacy-inventory-management

### Curam Contact page without the brand logo in the breadcrumb
![Screenshot 2023-09-06 at 4 39 21 PM](https://github.com/hlxsites/merative2/assets/1815714/daecb2c3-07a0-4e67-bf74-467c7c525cd0)

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/offers/contact-curam
- After (Changes from this PR): https://feat-breadcrumb--merative2--proeung.hlx.page/offers/contact-curam
  
## Testing Instruction

- Visit the test pages above and ensure that the breadcrumb renders correctly on all breakpoints.
- Ensure that if the breadcrumb image is set to `false` within the page metadata, the brand logo/image should be hidden.
